website/docs/getting-started/setup/instance-management/supervisor.md

```diff
@@ -3,28 +3,45 @@ sidebar_position: 1
 ---
 # Supervisor
 
-The container runs multiple processes, including the Appsmith server, Nginx, MongoDB, etc., inside a single Docker container. These processes are started and managed by [Supervisor](http://supervisord.org).
+The container runs multiple processes, including the Appsmith server, Nginx, MongoDB, etc., inside a single Docker container. These processes are started, managed, and monitored by Supervisor.
 
-Supervisor comes with a web interface for managing the various processes, available at [http://localhost/supervisor](http://localhost/supervisor), as well as a command-line interface towards the same goal.
+[Supervisor](http://supervisord.org/) provides:
+* [Web Interface](#web-interface)
+* [Command line Interface](#command-line-interface) 
 
-:::info
-The credentials for supervisor access are defined by the environment variables `APPSMITH_SUPERVISOR_USER` and `APPSMITH_SUPERVISOR_PASSWORD` in the `stacks/configuration/docker.env` file.
-:::
+## Web interface
+The Supervisor web interface can be accessed at `http://localhost/supervisor`.
```

Comment:
this line and the image ideally should be combine with step 3 so the user know they have to set it up first and then access it